Google.Api.Gax.Grpc.Tests/CallSettingsExtensionsTest.cs

```diff
@@ -177,7 +176,46 @@ public void WithEarlierDeadline_DeadlineIsEarlierThanExistingTimeout()
         }
 
         [Fact]
-        public void WithExpiration_NullExpiration()
+        public void WithRetry_NullInSettings_NullRetry()
```

Comment:
Thanks for these tests - it's only by pure fluke of me looking at Storage v2 that I picked up on this.